src/back-end/lib/db.ts

```diff
@@ -1270,14 +1270,6 @@ async function rawCWUProposalSlimToCWUProposalSlim(connection: Connection, raw:
 export const readManyCWUProposals = tryDb<[Id], CWUProposalSlim[]>(async (connection, id) => {
   const results = await connection<RawCWUProposalSlim>('cwuProposals as prop')
     .where({ opportunity: id })
-    .join<RawCWUProposalSlim>('cwuProposalStatuses as stat', function() {
```

Comment:
@dhruvio I've moved this logic out of the join and into a separate query below.  I noticed that since I added the `.whereNot` clause for the event changes in a previous PR, that this join was not returning all results.